include/bbtypes.h

```diff
@@ -152,6 +152,9 @@ typedef struct {
     /* 0x44 */ RecryptListEntry entries[0];
 } RecryptList;
 
+typedef   BSL_boolean;
+typedef BSL_boolean boolean;
```

Comment:
```suggestion
typedef enum {
    BSL_TRUE = 0,
    BSL_FALSE = 1
} BSL_boolean. boolean;
```
My STABS parser is not good at this sort of thing sorry

Should also probably go in `bb_nn.h`